community/kernel/src/main/java/org/neo4j/kernel/api/AccessMode.java

```diff
@@ -66,7 +66,29 @@ public boolean allowsSchemaWrites()
                 }
             },
 
-    /** Allows reading data and schema, but not writing. */
+    /** Allows writing data */
+    WRITE_ONLY
```

Comment:
This code is not strictly required by the current version, but will be needed in future, so we thought it simple enough, and small enough, to be OK to include now. If you have a strong feeling about this, we can remove it and add it back later.